packages/react-utilities/src/utils/getNativeElementProps.ts

```diff
@@ -65,3 +65,31 @@ export function getNativeElementProps<TAttributes extends React.HTMLAttributes<a
 
   return getNativeProps(props, allowedPropNames, excludedPropNames);
 }
+
+/**
+ * Splits the native props into ones that go to the 'root' slot, and ones that go to the 'primary' slot.
+ *
+ * This function is only for use with components that have a primary slot other than 'root'.
+ * Most components should use {@link getNativeElementProps} for their root slot if it is the primary slot.
+ *
+ * @returns An object containing the native props for the `root` and `primary` slots.
+ */
+export const getPartitionedNativeProps = ({
+  tagName,
+  props,
+  excludedPropNames,
+}: {
+  /** The element's tag name (e.g. 'div') */
```

Comment:
Which element?

Reply:
Good catch. It's confusing now that I've merged the root and primary slots to a single function. I'll update the comment, and even rename the arg to `primarySlotTagName` since it's important and to reduce confusion.